.npmignore

```diff
@@ -0,0 +1 @@
+*.story.js
```

Comment:
Why do we need this? If both, `.npmignore` and `.gitignore` are present, then it will use only the `.npmignore`! So this change undoes the exclusion done by `.gitignore`

>  In simpler terms, npm prefers the .npmignore file if it is there, but will fall back to the .gitignore file. So if both are present, it will use the .npmignore file.

Source: https://stackoverflow.com/a/24942436

This also means that we would be exposing any sensitive files excluded from `.gitignore` as we'd be bundling them into the package as mentioned in [For the love of god, don’t use .npmignore](https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d) whose title I found pretty funny.

I'd use the [`files`](https://docs.npmjs.com/files/package.json#files) field instead, which we seem to be using as well?

Reply:
Ah good point. I just wanted to ignore stories to be published as well

Reply:
I’m not even sure it works at the moment because of https://github.com/npm/npm/pull/11669. I’m on the phone now so I could not verify.